packages/link/src/lib/Link.tsx

```diff
@@ -0,0 +1,81 @@
+import { ReactNode, forwardRef } from 'react';
+import { Link as MuiLink, LinkProps as MuiLinkProps } from '@mui/material';
+import { SystemProps } from '@mui/system';
+import { OpenInNewIcon } from '@availity/mui-icon';
+
+// Scheme: https://tools.ietf.org/html/rfc3986#section-3.1
+// Absolute URL: https://tools.ietf.org/html/rfc3986#section-4.3
+const ABSOLUTE_URL_REGEX = /^[a-zA-Z][a-zA-Z\d+\-.]*?:/;
+
+// Windows paths like `c:\`
+const WINDOWS_PATH_REGEX = /^[a-zA-Z]:\\/;
+
+function isAbsoluteUrl(url: string) {
```

Comment:
We have very similar logic in the sdk. Is it worth updating the sdk and using that?

https://github.com/Availity/sdk-js/blob/master/packages/resolve-url/src/is-absolute-url.js